src/_guides/libraries/library-tour.md

```diff
@@ -919,6 +919,20 @@ also want to override the `==` operator. Objects that are equal (via
 `==`) must have identical hash codes. A hash code doesn’t have to be
 unique, but it should be well distributed.
 
+{{site.alert.tip}}
+  To consistently and easily implement the `hashCode` getter,
+  consider using the static hashing methods provided by the `Object` class.
+
+  For multiple objects, consider using [`Object.hash`][], 
+  while for collections
+  consider [`Object.hashAll`][] or [`Object.hashAllUnordered`][] 
+  according to your object's definition of equality. 
```

Comment:
I believe we use `()` after function names, so I added those. (They'd need to be added to the link definitions as well, of course.)

Also, I didn't understand the multiple objects vs. collections thing, and I wanted to reduce sentence length. So... Here's my attempt:

```suggestion
  To generate a single hash code for multiple objects,
  you can use [`Object.hash()`][].
  To generate a hash code for a collection,
  you can use either [`Object.hashAll()`][] (if element order matters)
  or [`Object.hashAllUnordered()`][].
```

I vaguely remember us talking about collection member terminology recently (_element? item? member?_) but I don't remember what we decided.

Reply:
These changes look good to me, thanks. I do believe we discussed element as the best choice here and it makes sense in this case as well.